doc/man1/genrsa.pod

```diff
@@ -112,6 +121,15 @@ Because key generation is a random process the time taken to generate a key
 may vary somewhat. But in general, more primes lead to less generation time
 of a key.
 
+For multi-prime RSA, which is defined in RFC 8017, unlike normal 2-prime RSA,
+it's security depends on both how many primes are used and how long the RSA
+modulus is. But it's still estimated by the difficulty of factoring the modulus.
+In simple terms, the more primes a multi-prime RSA key has, the easier for it
+being factored. Here is a link describing the security of multi-prime RSA
+generally: https://www.imperialviolet.org/2011/04/09/multiprime.html. And if
+you want to understand the very details, here is the right place to go to:
```

Comment:
s/you want to understand the very details, here is the right place to go to/you want to know more, this link provides the requisite details/